### DISCUSSION TITLE
Will they updated the music lbs

### DISCUSSION THREAD

**Discussion by Sbst397** [thatskymod/Sky-CotL-Scripts#128] [Q&A]
Well I know there is a music mod on Roman but it's not as good as the music lbs the violon, aurora voice, electrical guitar sounds very bad the interface is also rather small, the music mod was better they even added the grand piano and the cello it was a very good mod but it's been like a month I can't use I wondered if they given up on this mod, that's really sad because it's really the best for auto music play

**Comment by thatskymod** [comment]
The music lib (libsmusic.so) has been discontinued by its author and no lomger receives the support it once did. I appreciate you bringing this up, and have removed the lib from this collection.

As libRomanBeta is experiencing temporary outages due to the recent _Season of Moomin_ patch update, you can explore workarounds such as the [SkyAutoMusic](https://play.google.com/store/apps/details?id=com.zhukovartemvl.skyautomusic) app (Android) and [ThatSkyMod](https://github.com/XeTrinityz/ThatSkyMod) (PC) for instrument playback.
